src/msw/colordlg.cpp

```diff
@@ -76,11 +162,18 @@ wxColourDialogHookProc(HWND hwnd,
         CHOOSECOLOR *pCC = (CHOOSECOLOR *)lParam;
         wxColourDialog * const
             dialog = reinterpret_cast<wxColourDialog *>(pCC->lCustData);
+           
+        // Init the subclassing data.
+        g_subclassingData.originalProc = (WNDPROC)SetWindowLongPtr(hwnd, GWLP_WNDPROC, (LONG_PTR)wxColourDialogSubClassProc);
+        g_subclassingData.selectedColor = pCC->rgbResult;
+        g_subclassingData.colourDialog = dialog;
 
+        // Init the window title.
```

Comment:
Are the two newly added comments for existing code (lines 171 and 176) necessary ? The code seems quite self-documenting...